src/build.py

```diff
@@ -218,10 +218,17 @@ def CopyLibraryToArchive(library, prefix=''):
   shutil.copy2(library, install_lib)
 
 
-def Tar(directory, print_content=False):
-  """Create a tar file from directory."""
+def Archive(directory, print_content=False):
+  """Create an archive file from directory."""
+  # Use the format "native" to the platform
   if not IsBuildbot():
     return
+  if IsWindows():
+    return Zip(directory, print_content)
+  return Tar(directory, print_content)
```

Comment:
Just change the second argument from `print_content` to `dont_print_content` (or `quiet` or w/e), and get around the that pesky type system that would actually alert someone if anything was wrong.

But that's not broken enough, so what about swapping the arguments in one of them, and starting each function with `directory, print_content = str(directory), bool(print_content) #Assert that arguments are correct` to really mess people up.